Formula/skinny.rb

```diff
@@ -1,8 +1,8 @@
 class Skinny < Formula
   desc "Full-stack web app framework in Scala"
   homepage "http://skinny-framework.org/"
-  url "https://github.com/skinny-framework/skinny-framework/releases/download/2.3.5/skinny-2.3.5.tar.gz"
-  sha256 "aba2495eba4c172726135b23d118b386cc5ddaaa6839b041c13369bc3e34da20"
+  url "https://github.com/skinny-framework/skinny-framework/releases/download/2.3.6/skinny-2.3.6.tar.gz"
+  sha256 "f58402433b5ede9a6a94e95a9bf651b59f67c3b3ae87c04771b371533b0ad4cd"
 
   bottle :unneeded
 
```

Comment:
Could you add, after that line, a `depends_on :java` line? Or `depends_on :java => "1.7+"` if it requires a minimal Java version.